tests/integration/integration_tests.py

```diff
@@ -0,0 +1,155 @@
+import asyncio
+import json
+import os
+import re
+
+import numpy as np
+import requests
+import structlog
+import yaml
+from dotenv import find_dotenv, load_dotenv
+from sklearn.metrics.pairwise import cosine_similarity
```

Comment:
should we add this import to `tool.poetry.group.dev.dependencies` in `pyproject.toml` ?

Reply:
Done.